dynatrace/dynatrace.go

```diff
@@ -94,8 +109,7 @@ func (e *Exporter) ExportKindFor(*metric.Descriptor, aggregation.Kind) export.Ex
 
 // Export given CheckpointSet
 func (e *Exporter) Export(ctx context.Context, cs export.CheckpointSet) error {
-	// TODO tags
-	// TODO normalize
+	// TODO normalize. see mint package.
```

Comment:
FWIW not really a problem or blocker but we are not supposed to use mint name publicly. My fault it is already used in this package :/

Reply:
Yeah the internal abbreviation is not a name that would be helpful or descriptive for users/customers looking into the code. Not a blocker if just used in package names that are otherwise documented clearly but it certainly shouldn't go into the README or API docs.